### PR TITLE
feat LandingPage and routes

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**

### CODE REVIEW

**Comment by Shrinkhlarajpoot** [comment]
Feat.Landing page
-Added page routes
-Added Landing Page

https://codesandbox.io/s/github/Shrinkhlarajpoot/NotesApp/tree/landingPage

**Review by KaustubhM1997 — APPROVED** [review]
Everything looks good!

File: src/utils/FormValidationChecker.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  if (password && !password.match(passwordFormat)) {
+    err.password =
+      "Too weak!!Should contain one character one digit and length between 4 and 8";
+  }
+  if (!confirmpassword) {
+    err.confirmpassword = "Enter confirm password";
+  }
+  if (confirmpassword && password !== confirmpassword) {
+    err.passwordnotmatch = "Password didn't match";
+  }
```
Comment (gopalbbharadva) [inline src/utils/FormValidationChecker.jsx:33]: Suggestion : You can check for && as no else.

**Review by gopalbbharadva — APPROVED** [review]
Code is well structured, just remove consoles from API calls, generally we all forgot to remove it from API calls.

**Pull request merged**